sig-architecture/production-readiness.md

```diff
@@ -32,18 +32,24 @@ the PRR. The template points out these as needed.
 
 ## Submitting a KEP for production readiness approval
 
+The KEP template production readiness questionnaire should be filled out by the KEP authors, and this will be reviewed by the SIG leads. Once the leads are satisfied with both the overall KEP (i.e., it is ready to move to `implementable` state) and the PRR answers, the authors may request PRR approval.
+
```

Comment:
@npolshakova, we've two duplicate paras for the same text. Could you please remove one. Thanks!